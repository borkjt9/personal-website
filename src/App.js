import React from 'react';
// Normalize.css for additional detail, see https://necolas.github.io/normalize.css/
import 'normalize.css';
import './App.scss';
import './styles/theme/typography.scss';
import './styles/theme/spacing.scss';

import './styles/base.scss';
import './styles/animations.scss';
import './styles/transitions.scss';


import LandingPage from './landing-page/landing-page';

const App = () => (
  <div className="App">
    <LandingPage />
  </div>
);

export default App;
