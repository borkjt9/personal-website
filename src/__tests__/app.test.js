import React from 'react';
import Enzyme, { shallow } from 'enzyme';

import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import App from '../jest_examples/app';

Enzyme.configure({ adapter: new Adapter() });

describe('App Jest Example', () => {
  it('contains React text', () => {
    const component = shallow(<App />);
    expect(component.text()).toContain('React Simple Starter');
    // expect(component).toContain('john borkowski')
  });
  it('should render correctly', () => {
    const output = shallow(<App />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
