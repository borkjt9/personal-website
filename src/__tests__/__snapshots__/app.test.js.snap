// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Jest Example renders correctl 1`] = `
<div>
  React Simple Starter
</div>
`;
